R/checks.R

```diff
@@ -207,6 +207,32 @@ check_form4.1_box5 <- function(form) {
                         "Textfield and coding box 5 of form4.1 is completed if and only if the other option is chosen"))
 }
 
+#' Filters out invalid rows for box 6 of form 4.1 for location of first administration = At The Injury Scene
```

Comment:
Time from injury to antibiotics administration must be within 5 hours from injury and less than time from injury to hospital administration

Reply:
Don't copy the comment structure for check_formx_boxn. They are for checking coding boxes. Refer to check_injdate_hspdate instead.